webrtc/RTCRtpParameters-degrationPreference.html

```diff
@@ -0,0 +1,87 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>RTCRtpParameters degrationPreference</title>
```

Comment:
typo: `degrationPreference`=> `degradationPreference`

(The filename needs to be updated as well)